tests/data/fxa-server-metrics.yaml

```diff
@@ -0,0 +1,21 @@
+# Any copyright is dedicated to the Public Domain.
+# https://creativecommons.org/publicdomain/zero/1.0/
+
+---
+$schema: moz://mozilla.org/schemas/glean/metrics/2-0-0
+
+event:
```

Comment:
Ooooooh, I was a bit thrown off by this naming. It's actually a string metric inside.

Reply:
@badboy would `fxa-server-string-metric.yaml` be preferred?

Reply:
Nah, it's fine this way.